### PR TITLE
Network tables

### PR DESCRIPTION
**Pull request by ounsworth**
From Finlay: "Just merge the latest version of my branch into matt_mat. I made some important fixes"

### CODE REVIEW

File: src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java [hunk 1]
```diff
@@ -3,43 +3,39 @@
...
+				System.out.println("fps: "+visionData.fps);
+				try {
+					Thread.sleep(10);
+				} catch (InterruptedException e) {
+					e.printStackTrace();
+				}
+			} catch (Exception e) {
 				e.printStackTrace();
 			}
 		}
```
Comment (ounsworth) [inline src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java]: I thought I renamed this file yesterday?

File: src/ca/team2706/vision/trackerboxreloaded/Main.java [hunk 2]
```diff
@@ -51,17 +50,19 @@
...
 		 */
 		public void encode(NetworkTable table) {
-			// TODO: add code
-			// basic networktable code: table.getEntry("data").setString("foo);
-			table.getEntry("data").setString(DataUtils.encodeData(data));
+			// basic networktable code: table.getEntry("data").forceSetString("foo);
+			table.getEntry("fps").forceSetDouble(fps);
+			table.getEntry("ctrX").forceSetDouble(ctrX);
+			table.getEntry("numTargetsFound").forceSetDouble(numTargetsFound);
 		}
```
Comment (ounsworth) [inline src/ca/team2706/vision/trackerboxreloaded/Main.java]: I thought I took out encode() and decode() yesterday to simplify things?

File: src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java (hunk 1 above)
Comment (thatmadhacker) [reply src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java]: That was on Matt's branch I think.
At 230 today I will  fix all da conflicts

Comment (ounsworth) [reply src/ca/team2706/vision/trackerboxreloaded/ExampleServer.java]: Ok, that makes sense. I suggest you:

`matt_math` --merge--> `NetworkTables`
fix conflicts, build, test
`NetworkTables` --pull request--> `matt_math`

**Pull request closed without merging**